src/core/useRcbPlugin.ts

```diff
@@ -17,7 +18,7 @@ import { getValidator } from "../utils/getValidator";
 /**
  * Plugin hook that handles all the core logic.
  *
- * @param pluginConfig configurations for the plugin
+ * @param pluginConfig Configurations for the plugin.
```

Comment:
Unnecessary comment changes, there're some other ones in the rest of the code, can revert these changes :3